frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/mobile_toolbar_v3/appflowy_mobile_toolbar.dart

```diff
@@ -179,7 +180,13 @@ class _MobileToolbarState extends State<_MobileToolbar>
   //  but in this case, we don't want to update the cached keyboard height.
   // This is because we want to keep the same height when the menu is shown.
   bool canUpdateCachedKeyboardHeight = true;
-  ValueNotifier<double> cachedKeyboardHeight = ValueNotifier(0.0);
+
+  /// when the [_MobileToolbar] disposed before the keyboard height can be updated in time,
+  /// there will be an issue with the height being 0
+  /// this is used to globally record the height.
+  static double _maxKeyboardHeight = 0.0;
+  ValueNotifier<double> cachedKeyboardHeight =
```

Comment:
If `_maxKeyboardHeight` is used as the initial value every time, `cachedKeyboardHeight` will never be 0 after the first initialization. I assume this might have side effects on other logic, such as closing the keyboard.

Reply:
`cachedKeyboardHeight` is used to keep the same height when the menu is shown, and it was controlled by `canUpdateCachedKeyboardHeight`, after `_MobileToolbar` disposed and recreated a new one, `cachedKeyboardHeight` will be set to 0, that will cause the issue —— #7049， and using `_globalCachedKeyboardHeight` can prevent the issue caused by its rapid re-initialization before the height can be obtained